AutomationHelpers/src/UserCodeCollections/FileLibrary.cs

```diff
@@ -43,6 +43,18 @@ public static void WriteToFile(string text, string filenamePrefix, string fileEx
                 Console.WriteLine(ex.ToString());
             }
         }
+		
+		// <summary>
+        /// Open an existing file and add a new line of text.
+        /// </summary>
+        /// <param name="text">Text that the file should contain</param>
+        /// <param name="path">Full qualified path to load the file including filename and extension</param>
+        [UserCodeMethod]
+        public static void AppendToExistingFile(string text, string path)
```

Comment:
Rather call "AppendLineToExistingFile" because you explicitly add a NewLine?
Frankly, I would not add the NewLine before the text

Reply:
Renamed.
Because in my case the line break is needed i add a parameter which let the caller decide if the line break should be added.